README.md

```diff
@@ -1,5 +1,7 @@
 # Krkn aka Kraken
 ![Workflow-Status](https://github.com/krkn-chaos/krkn/actions/workflows/docker-image.yml/badge.svg)
+![coverage](https://krkn-chaos.github.io/krkn-lib-docs/coverage_badge_krkn.svg)
```

Comment:
Coverage seems to off: https://github.com/krkn-chaos/krkn/blob/40b381a95b522cad5816e7674764f2dde7c950fb/README.md